pages/WatchList.js

```
@@ -1,94 +1,70 @@
+            onCancelButtonPress: () => {
+              setSearchQuery('');
+            },
+          }
+        : undefined,
+    });
+  }, [navigation, showWatchList, handleSearch]);
+
   useEffect(() => {
     getAccountAndSession();
-  }, [refreshIndicator]);
+  }, [getAccountAndSession]);
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)